README.md

```diff
@@ -36,10 +36,17 @@ import alias from 'rollup-plugin-alias';
 export default {
   input: './src/index.js',
   plugins: [alias({
-    somelibrary: './mylocallibrary'
+    resolve: ['.jsx', '.js'],
+    entries:[
+      {find:'somelibrary', replacement: './mylocallibrary'},
+      {find:^./foobar\/path.*/i, replacement: './bar', isRegEx:true} 
```

Comment:
I think there is a leading `/` missing in the RegExp. Also, is the flag really necessary? E.g. in the code, we could just check if `find instanceof RegExp`, which works nicely for literals.

Reply:
Fixed, also switched to use instanceof RegEx instead of an needless option